tests/test_skills/test_error.py

```diff
@@ -63,40 +66,40 @@ class TestSkillError:
 
     def setup(self):
         """Test the initialisation of the AEA."""
-        cls = self
-        cls.node = LocalNode()
```

Comment:
was the node not needed?

Reply:
I did not found any reference to node in test code. also tests work without node, so looks node definition just copy/pasted.